gnomad_qc/v4/create_release/create_release_sites_ht_genomes.py

```diff
@@ -0,0 +1,50 @@
+"""script to create release sites ht for v4 genomes."""
+import argparse
+import logging
+
+import hail as hl
+from gnomad.resources.grch38.reference_data import dbsnp
+
+from gnomad_qc.v4.resources.annotations import get_vep
+from gnomad_qc.v4.resources.constants import CURRENT_VERSION
+
+logging.basicConfig(
+    format="%(asctime)s (%(name)s %(lineno)s): %(message)s",
+    datefmt="%m/%d/%Y %I:%M:%S %p",
+)
+logger = logging.getLogger("create_release_ht")
+logger.setLevel(logging.INFO)
+
+
+def remove_missing_vep_fields(vep_ht: hl.Table) -> hl.Table:
+    """
+    Remove fields from VEP 105 annotations that are missing in all rows.
+
+    :param vep_ht: Table containing VEP 105 annotations
+    :return: Table containing VEP 105 annotations with missing fields removed
+    """
+    vep_ht = vep_ht.annotate(vep=vep_ht.vep.drop("colocated_variants", "context"))
+
+    vep_ht = vep_ht.annotate(
+        vep=vep_ht.vep.annotate(
+            transcript_consequences=vep_ht.vep.transcript_consequences.map(
+                lambda x: x.drop("minimised", "swissprot", "trembl", "uniparc")
+            )
+        )
+    )
+
+    for consequence in [
+        "intergenic_consequences",
+        "motif_feature_consequences",
+        "regulatory_feature_consequences",
+    ]:
+        vep_ht = vep_ht.annotate(
+            vep=vep_ht.vep.annotate(
+                **{
+                    consequence: vep_ht.vep[consequence].map(
+                        lambda x: x.drop("minimised")
+                    )
+                }
+            )
+        )
+    return vep_ht
```

Comment:
A suggestion for readability and code reuse,  since you aren't doing anything to the actual ht, you could just pass the ht vep expression, vep_expr, to the entire function and return that instead of the entire table, I think that would read the best.   
```suggestion
    vep_expr = vep_expr.drop("colocated_variants", "context")

    vep_expr = vep_expr.annotate(
            transcript_consequences=vep_expr.transcript_consequences.map(
                lambda x: x.drop("minimised", "swissprot", "trembl", "uniparc")
            )

    for consequence in [
        "intergenic_consequences",
        "motif_feature_consequences",
        "regulatory_feature_consequences",
    ]:
        vep_expr = vep_expr.annotate(
                **{
                    consequence: vep_expr[consequence].map(
                        lambda x: x.drop("minimised")
                    )
                }
            )
    return vep_expr
```

Reply:
I made this suggestion here so havent had a chance to format it, it may fail black if you accept it. It will also require updates to the docs.